hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/persistence/AbstractReconSqlDBTest.java

```diff
@@ -115,7 +115,7 @@ protected void configure() {
   public void createSchema(Injector inj) {
     ReconSchemaManager reconSchemaManager =
         inj.getInstance(ReconSchemaManager.class);
-    reconSchemaManager.createReconSchema();
+    reconSchemaManager.createAndUpgradeReconSchema();
```

Comment:
Can we add some more test cases related to error and failure and assert when and all schema version gets inserted/ updated in schema version table ?